mmseg/models/backbones/beit.py

```diff
@@ -57,12 +61,16 @@ def __init__(self,
         self.num_heads = num_heads
         head_embed_dims = embed_dims // num_heads
         self.scale = qk_scale or head_embed_dims**-0.5
-        if qv_bias:
-            self.q_bias = nn.Parameter(torch.zeros(embed_dims))
-            self.v_bias = nn.Parameter(torch.zeros(embed_dims))
-        else:
-            self.q_bias = None
-            self.v_bias = None
+
+        self.q_bias = nn.Parameter(
+            torch.zeros(embed_dims)) if q_bias else torch.zeros(
+                embed_dims, requires_grad=False)
+        self.k_bias = nn.Parameter(
+            torch.zeros(embed_dims)) if k_bias else torch.zeros(
+                embed_dims, requires_grad=False)
+        self.v_bias = nn.Parameter(
+            torch.zeros(embed_dims)) if v_bias else torch.zeros(
+                embed_dims, requires_grad=False)
```

Comment:
might register buffer for q_bias, k_bias, v_bias, as they will not be moved to GPU when calling `model.cuda()` and will not be saved in state_dict